src/store.js

```diff
@@ -1,10 +1,85 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from "vue";
+import Vuex from "vuex";
 
-Vue.use(Vuex)
+Vue.use(Vuex);
 
 export default new Vuex.Store({
-  state: {},
-  mutations: {},
+  state: {
+    acceptedLegal: false,
+    selectedLayer:null,
+    wmsLayers: [],
+    compareLayers: []
+  },
+  mutations: {
+    ADD_COMPARE_LAYER(state, newLayer) {
```

Comment:
It's optional to use the uppercase/snakecase way of writing these mutations.(https://vuex.vuejs.org/guide/mutations.html#using-constants-for-mutation-types) 

But we never use it like this in other projects (maybe voorhoede does, i don't know). But i don't like how it looks inside the project where everything else is camelCase ?